### ISSUE TITLE
Support to show several states

### ISSUE THREAD

**Issue by kalkih** [kalkih/mini-graph-card#33]
@ideasman69 
> How hard is it to have the option of adding in the current value to the right of the screen? Like over here:
> 
> ![image](https://user-images.githubusercontent.com/19260137/52098583-b55d1d00-2623-11e9-951a-c52a7ec825dd.png)
> 
> I suppose it can get tricky with more than 2 sensors

**Comment by kalkih** [comment]
Definitely want to add this functionality, possibly with a `show_state` function in the entity object.
Have to figure out how to handle a few things, like default state.

Could look something like this.

```yaml
# Example of future option
- type: custom:mini-graph-card
  entities:
    - entity: sensor.example
      show_state: true
    - entity: sensor.example2
      show_state: true
    - sensor.example2
```

**Comment by kalkih** [comment]
Now available through the entity option `show_state` in  [v0.2.1](https://github.com/kalkih/mini-graph-card/releases/tag/v0.2.1)

**Comment by ideasman69** [comment]
Thanks @kalkih - looks good, but a couple of suggestions :blush:

When having two states, any chance of having the same font size as the main one on the left?
![image](https://user-images.githubusercontent.com/19260137/52187541-2dc11980-2882-11e9-8cbd-3ddd6ab73aff.png)

when using 3 states, would it be an option to have all 3 states the same at the same size, across the top like this:

![image](https://user-images.githubusercontent.com/19260137/52187568-5cd78b00-2882-11e9-9c49-c131edb9657c.png)

Like one left justified, one middle justified and the last right justified.

**Comment by kalkih** [comment]
I don't think so, unfortunately.
That was actually similar to the design I came up with first, and while it looked great and worked well in specific cases, like yours above, it just didn't work in many others. For example when the card was narrower, when there were more than 2-3 visible states or when the states / unit of measurements were longer.

In the end I opted to go with a design that worked well with any number of additional states and card widths.

However, nothing is set in stone and it's possible that I could add something like that as an option in the future, if I come up with a good execution.

OT, but I see that the graph legend looks odd in your screenshots, the colored squares and the text doesn't line up quite right, fix inc!
